user-interface/package.json

```diff
@@ -15,24 +15,24 @@
     "@types/jsdom": "^21.1.7",
     "@types/node": "^22.7.5",
     "@types/react": "^18.3.11",
-    "@types/react-dom": "^18.3.0",
+    "@types/react-dom": "^18.3.1",
     "@uswds/uswds": "^3.9.0",
     "dotenv": "^16.4.5",
     "launchdarkly-react-client-sdk": "^3.4.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-router-dom": "^6.26.2",
+    "react-router-dom": "^6.27.0",
     "react-select": "^5.8.1"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^6.5.0",
+    "@testing-library/jest-dom": ">6.4.8 <6.6.0 || >6.6.0 <7.0.0",
```

Comment:
There is [an issue](https://github.com/testing-library/jest-dom/issues/641) with an open PR to fix a problem with version 6.6.0.